.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "grupovama"]
 	path = grupovama
 	url = git@git.vauxoo.com:vauxoo/grupovama.git
-	branch = 17.0
+	branch = 17.0-grupovama-permissions-hernan-85405
```

Comment:
This change is not required, you can update the sha without changing this, BTW, groupovama submodule is not update with last changes, It is pointing to another commit